packages/adwaita_icon_theme.rb

```diff
@@ -23,9 +23,9 @@ class Adwaita_icon_theme < Meson
   depends_on 'vala' => :build
   depends_on 'xdg_base'
 
+  gnome
+
   def self.postinstall
-    # Update mime database.
-    system "update-mime-database #{CREW_PREFIX}/share/mime"
     # This is skipped when DESTDIR is set during build.
     system "gtk-update-icon-cache -qtf #{CREW_PREFIX}/share/icons/Adwaita" if File.file?("#{CREW_PREFIX}/bin/gtk-update-icon-cache")
```

Comment:
It looks like we can probably drop this-- `lib/gnome.rb` runs `gtk-update-icon-cache` on `CREW_PREFIX/share/icons` which should cover `CREW_PREFIX/share/icons/Adwaita`.

Reply:
done